app/templates/flat-page.hbs

```diff
@@ -20,17 +20,17 @@
               <div class="share-tools clearfix">
                 <ul class="sharelinks clearfix">
                   <li class="fa-hover facebook">
-                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" type="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
+                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" aria-label="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
```

Comment:
One more thing though. You want the text alternative to be a useful description of the action, so "Share on Facebook" is more useful that just "Facebook". same applies to the other two share actions. Here's the a11y slide deck for some good review: https://docs.google.com/presentation/d/1qCg2VjLrr6iuD6qxTU_8rYLA5vaR_OY6WHjCf9P00Xk/edit#slide=id.g52a3691329_0_0